### PR TITLE
docs(K8s): stubs for Kubernetes v2 docs

### PR DESCRIPTION
**Pull request by dorbin**
Just some stubs for you to get started with, Lars

### CODE REVIEW

**Review by lwander — COMMENTED** [review]
Can you change the file names to treat "kubernetes" consistently? I'm not sure what the existing convention is, but these seem to be mixed between `k8s` and `kub`

**Comment by dorbin** [comment]
Yes, but there is already "kub" and "kubernetes" before this change. Should I just change me "k8s"s to "kub"s and let the existing inconsistencies stand?

**Comment by lwander** [comment]
Ah shame, let's leave the existing stuff as-is and replace k8s with kub

File: reference/providers/k8sv2/index.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+ Layout: single
```
Comment (lwander) [inline reference/providers/k8sv2/index.md]: This directory seems to follow the k8s naming rather than kub

**Review by lwander — APPROVED** [review]
Thanks!

**Comment by lwander** [comment]
(Just waiting for the build)

**Pull request merged**